### PR TITLE
fix:Add white-space style

### PR DESCRIPTION
**Pull request by gee05053**
This PR resolves https://github.com/lablup/backend.ai-webui/issues/2084

Before
스크린샷 2023-12-11 오후 7 18 38

After
스크린샷 2023-12-11 오후 7 17 39

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Review by yomybaby — CHANGES_REQUESTED** [review]
In this case, use interpolation
- lit-translate: https://github.com/stefanholzapfel/lit-translate#interpolation
- i18n-react: https://www.i18next.com/translation-function/interpolation

**Review by lizable — APPROVED** [review]
LGTM.

File: resources/i18n/de.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "webui": {
     "menu": {
-      "WelcomeMessage": "Herzlich willkommen,",
+      "WelcomeMessage": "Willkommen, {{ userName }}",
       "WelcomeMessage_2": " .",
```
Comment (yomybaby) [inline resources/i18n/de.json]: Is there any reason to keep the `WelcomeMessage_2`? You can put `WelcomeMessage_2` to the `WelcomeMessage` such as `.` and `님`.

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: resources/i18n/de.json (hunk 1 above)
Comment (gee05053) [reply resources/i18n/de.json]: Sorry. I deleted WelcomeMessage2 but I forgot commit that. I will commit right now

File: src/components/backend-ai-webui.ts [hunk 2]
```diff
@@ -1862,9 +1862,9 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                 </div>
                 <div slot="navigationIcon" class="vertical-line" style="height:20px;margin:0;"></div>
                 <div class="horizontal center layout" slot="title" id="welcome-message" style="font-size:12px;margin-left:10px;" class="draggable">
-                  <p>${_t('webui.menu.WelcomeMessage')}</p>
-                  <p class="user-name">${this._getUsername()}</p>
```
Comment (yomybaby) [inline src/components/backend-ai-webui.ts:1866]: please check  https://github.com/lablup/backend.ai-webui/pull/1008/commits/94bfc08edeaedca389560d4d4726f66cb9ddea80 commit related to `user-name` class.

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-webui.ts [hunk 3]
```diff
@@ -1862,9 +1863,12 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                 </div>
                 <div slot="navigationIcon" class="vertical-line" style="height:20px;margin:0;"></div>
                 <div class="horizontal center layout" slot="title" id="welcome-message" style="font-size:12px;margin-left:10px;" class="draggable">
-                  <p>${_t('webui.menu.WelcomeMessage', {
-                    userName: this._getUsername(),
-                  })}</p>
+                  <p style="display:flex;align-items:center;white-space:pre-wrap;">${_tr(
+                    'webui.menu.WelcomeMessage',
+                    {
+                      userName: `<p class="user-name">fdsfmsdklfnsklfnlksnfkslnflkknllnnkl</p>`,
```
Comment (yomybaby) [inline src/components/backend-ai-webui.ts]: Please check this part.
`fdsfmsdklfnsklfnlksnfkslnflkknllnnkl`

File: src/components/backend-ai-webui.ts [hunk 4]
```diff
@@ -51,6 +51,7 @@ import {
   registerTranslateConfig,
   translate as _t,
   use as setLanguage,
+  translateUnsafeHTML as _tr,
```
Comment (yomybaby) [inline src/components/backend-ai-webui.ts]: Is there a lit-translate option to keep specific HTML like`transKeepBasicHtmlNodesFor` option of `react-i18next`?
Allowing all HTML is not safe.

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-webui.ts (hunk 4 above)
Comment (gee05053) [reply src/components/backend-ai-webui.ts:54]: @yomybaby 
There is no option..
But I found that I can customize registerTranslateConfig()'s interplate option and I think that is work well.
I will apply and request review again

Comment (gee05053) [reply src/components/backend-ai-webui.ts]: @yomybaby 
I've searched for that feature, but lit-translate doesn't support it.
Therefore, I solved it using the method we discussed in the meeting

**Review by yomybaby — CHANGES_REQUESTED** [review]
Please check again. It looks like this when the browser width is around 1100px.
image

**Comment by gee05053** [comment]
@yomybaby 
I forgot to delete test style,,
I delete that and UserDropdownMenu component is applied user-name style too.

File: react/src/components/UserDropdownMenu.tsx [hunk 5]
```diff
@@ -55,6 +55,10 @@ const UserDropdownMenu: React.FC = () => {
       style: {
         color: '#1f1f1f',
         cursor: 'default',
+        maxWidth: '15vw',
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
```
Comment (yomybaby) [inline react/src/components/UserDropdownMenu.tsx]: How about applying this using the `ellipsis` prop of `Typhora.Text`?

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: react/src/components/UserDropdownMenu.tsx (hunk 5 above)
Comment (yomybaby) [reply react/src/components/UserDropdownMenu.tsx]: If possible, setting style without explicit size(`15vw`) and color is better for maintenance.

**Review by yomybaby — APPROVED** [review]
I just push a commit. Please check this.
Thak you for contribution.
LGTM!

**Pull request merged**